src/index.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './components/App/App.jsx';
+
+const root = <div id="root">1</div>;
+
+ReactDOM.render(root, document.body);
+
+ReactDOM.render(<App />, document.getElementById('root'));
```

Comment:
...and there we need to provide to render method that element that you have created

Reply:
You already have that element. You haven't to call getElementById
```suggestion
ReactDOM.render(<App />, rootElement);
```